c2corg_api/search/mappings/outing_mapping.py

```diff
@@ -57,17 +63,59 @@ class Meta(BaseMeta):
         enum_mapper=sortable_glacier_ratings)
     avalanche_signs = QEnumArray(
         'avdate', model_field=Outing.avalanche_signs)
+    ski_rating = QEnumRange(
+        'trat', model_field=Outing.ski_rating,
+        enum_mapper=sortable_ski_ratings)
+    labande_global_rating = QEnumRange(
+        'srat', model_field=Outing.global_rating,
```

Comment:
For routes we rather use ``lrat``, see https://github.com/c2corg/v6_api/blob/master/c2corg_api/search/mappings/route_mapping.py#L64

And the model_field should be
```
model_field=Route.labande_global_rating
```